plans/convert.pp

```diff
@@ -13,6 +13,9 @@
   # Common Configuration
   String                            $compiler_pool_address = $master_host,
   Array[String]                     $dns_alt_names         = [ ],
+
+  # Options
+  Boolean                           $configure_node_groups = true,
```

Comment:
I think the $configure_node_groups should be dynamically generated rather than relying on a human.  Create a task to find the value of PE version in order to make the boolean.

Reply:
> I think the $configure_node_groups should be dynamically generated rather than relying on a human. Create a task to find the value of PE version in order to make the boolean.

I think that option not to create additional classification is very useful. There is no need for it in standard configuration with only primary and replica. Also module doesn't provide a plan to promote a replica and one would have to remove classifications added by the module to use standard promote procedure